tests: net: socket: af_packet_ipproto_raw move to new ztest API

hunterhu3000 opened the pull request:
Move af_packet_ipproto_raw tests to use new ztest API
TEST=twister -T tests/net/socket/af_packet_ipproto_raw/


Signed-off-by: Hu Zhenyu <zhenyu.hu@intel.com>

nashif commented on tests/net/socket/af_packet_ipproto_raw/src/main.c:
```diff
@@ -107,9 +107,11 @@ void test_setup(void)
 }
 
-void test_sckt_raw_packet_raw_ip(void)
+ZTEST(sckt_packet_raw_ip_test_suite, test_sckt_raw_packet_raw_ip)
```
no need for _test_suite here.

hunterhu3000 replied:
> no need for _test_suite here.

Updated, thanks to review

hunterhu3000 committed:
tests: net: socket: af_packet_ipproto_raw move to new ztest API

Move af_packet_ipproto_raw tests to use new ztest API
TEST=twister -T tests/net/socket/af_packet_ipproto_raw/

Signed-off-by: Hu Zhenyu <zhenyu.hu@intel.com>

hunterhu3000 commented:
This test case cannot be run with CONFIG_ZTEST_SHUFFLE =y, as some of the test case cannot be run twice.
